Support/hi.lproj/Localizable.strings

```diff
@@ -0,0 +1,76 @@
+// Messages for Hindi (हिन्दी)
+// Exported from translatewiki.net
+// Author: Abijeet Patro
+
```

Comment:
`library_refresh_error.retrieve.description` 
`library_refresh_error.parse.description`
`library_refresh_error.process.description`

Reply:
@translatewiki Please double check all strings for Hindi language.

Reply:
> `library_refresh_error.retrieve.description` 
> `library_refresh_error.parse.description`
> `library_refresh_error.process.description`

what is the problem?

Reply:
> @translatewiki Please double check all strings for Hindi language.

what is the problem?